src/client.cc

```
@@ -134,60 +134,101 @@ struct ScopedRDMARegistration {
-  this->resp_.contents.push_back(Item(std::move(err)));
-  this->itr_ = resp_.contents.begin();
+  resp_ = std::make_shared<ListObjectsResponse>();
+  resp_->contents.push_back(Item(std::move(err)));
+  itr_ = resp_->contents.begin();
 }
 
 ListObjectsResult::ListObjectsResult(Client* const client,
                                      const ListObjectsArgs& args)
     : client_(client), args_(args) {
-  Populate();
+  resp_ = std::make_shared<ListObjectsResponse>();
```

Cppcheck (2.21.0)

[performance] Variable 'resp_' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)